### ISSUE TITLE
Crash on macOS after entering password to enter wallet

### ISSUE THREAD

**Issue by rex4539** [monero-project/monero-gui#1122]
git @ bd173c9 (or later)

Reproducibility: always

Steps:
1. Launch GUI.
2. Enter password and hit enter.

What happened:
```
Error: memset_s failed
Process 99924 exited with status = 1 (0x00000001)
```

Expected result:
GUI does not crash.

Notes:
Looks like the crash was introduced sometime after git @ bd173c9
For example, git @ 4bf0bef does not crash.

**Comment by leonklingele** [comment]
https://github.com/monero-project/monero/pull/3289 should fix it

**Comment by rex4539** [comment]
Fixed with git @ 084c1c8
